README.md

```diff
@@ -45,13 +54,246 @@ Each part of Data Driven Testing is designed to handle a specific ICU version.
+   are executed.
+
+1. Next, get the code and data from the [Unicode Conformance GitHub
+   site](https://github.com/unicode-org/conformance).
+
+1. From the green Code button on Git but, select one of the options for cloning the
+project, or download the project as a ZIP file.
+
+1. Unzip if needed. Then make sure that the directory contains directories for testdriver, testgen, schema, executors, verifier.
+
+
+### Set up tools and execution environments
+Some setup is requirel to run Data Driven Testing code locally:
+
+- Install the Python package `jsonschema`
+    * In a standard Python environment, you can run
+        ```
+        pip install jsonschema
+        ```
+    * Some operating systems (ex: Debian) might prefer that you install
+    the OS package that encapsulates the Python package
+        ```
+        sudo apt-get install python-jsonschema
+        ```
+
+Note: Setting up Python modules or other software may be better done using a
+virtual environment such as using `pip env` for installation.
+
+- Install the minimum version supported by ICU4X
+    * The latest minimum supported supported Rust version ("MSRV") can be found in the
+    [`rust-toolchain.toml` file](https://github.com/unicode-org/icu4x/blob/main/rust-toolchain.toml)
+    * To view your current default Rust version (and other locally installed Rust versions):
+        ```
+        rustup show
+        ```
+    * To update to the latest Rust version:
+        ```
+        rustup update
+        ```
+  - Install `logrotate`
+      ```
+      sudo apt-get install logrotate
+      ```
+
+### Running End-To-End on a local computer
+
+Running conformance is easy. Simply execute the script that first generates
+data, runs executors, checks data against schema, and then runs the verifier,
+creating the dashboard.
+
+The standard script runs all components on all platforms with all ICU
+version. Note that this takes a few minutes.
+
+`` bash generateDataAndRun.sh ``
+
+Output will be created in the directory TEMP_DATA.
+
+To run a quicker version that uses only 100 test cases for each instance, run this:
+
+``
+bash genData100.sh
+``
+which does the same thing much faster, resulting in the directory TEMP_DATA_100.
+
+### Viewing conformance results in a browser
+HTML output is found in the subdirectory testReports. To visualize this on your
+computer, start a webserver. For example:
+
+``
+python3 -m http.server 9000 &
+``
+
+Then open the file testResults/index.html under the testReports folder in either
+TEMP_DATA or TEMP_DATA_100 or in a custom folder.
+
+
+### How this works
+
+Two main pieces are used to create conformance output:
+* run_config.json which describes the platforms and components to be executed
+
+* execution scripts that run all the steps of conformance testing. This file
+  extracts configurations from run_config
+  
+  * generateDataAndRun.sh runs tests across all platforms, components, and ICU
```

Comment:
if this is not a sub-bullet, then unindent to be flush with the other points. if this is meant to indent, then indent by 4 spaces to make Markdown parsers happy